pkg/cmd/roachtest/create_index.go

```diff
@@ -23,13 +23,13 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 )
 
-func registerCreateIndexTPCC(r *registry) {
+func registerSchemaChangeTPCC(r *registry) {
```

Comment:
Feels like this test should be moved to `schemachange.go`.

Reply:
Done. Added a commit to move the test.